projects/libjpeg-turbo/libjpeg_turbo_fuzzer.cc

```diff
@@ -39,10 +39,28 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
         return 0;
     }
 
-    std::unique_ptr<unsigned char[]> buf(new unsigned char[width * height * 3]);
+    const int buffer_size = width * height * 3;
+    std::unique_ptr<unsigned char[]> buf(new unsigned char[buffer_size]);
     tjDecompress2(
         jpegDecompressor, data, size, buf.get(), width, 0, height, TJPF_RGB, 0);
 
+    // For memory sanitizer, test each output byte
```

Comment:
It is indeed bad that MSAN doesn't complain about this automatically (https://github.com/google/sanitizers/issues/883)
That being said, you want `__msan_check_mem_is_initialized()`.